docs/defi/fcm/index.md

```diff
@@ -0,0 +1,184 @@
+
+### The Capital Flow
+
+```mermaid
+sequenceDiagram
+    participant User
+    participant ALP
+    participant MOET
+    participant FYV
+    participant Yield
+
+    User->>ALP: 1. Deposit FLOW collateral
+    ALP->>ALP: 2. Calculate borrowing capacity
+    ALP->>MOET: 3. Auto-borrow MOET
+    MOET->>FYV: 4. Deploy to yield strategy
+    FYV->>Yield: 5. Generate returns
+    Yield->>FYV: 6. Accumulate yield
+
+    Note over ALP,FYV: When collateral price drops:
+    FYV->>ALP: 7. Provide funds for rebalancing
+    ALP->>MOET: 8. Repay debt automatically
+
+    Note over User,ALP: Result: Position stays healthy
+```
+
+### Step-by-Step Flow
+
+1. **User deposits collateral** (e.g., FLOW tokens) into an ALP position
+2. **ALP auto-borrows** MOET against the collateral to reach target health ratio (1.3)
+3. **Borrowed MOET flows** to a FYV strategy (via DrawDownSink)
+4. **FYV deploys capital** into yield-generating opportunities (farms, LPs, etc.)
+5. **Yield accumulates** and compounds automatically
+6. **If collateral price drops**: FYV provides liquidity to ALP (via TopUpSource)
+7. **ALP repays debt** automatically to prevent liquidation
+8. **User keeps position healthy** without manual intervention
+
+## Key Innovations
+
+### 1. Yield-Powered Liquidation Prevention
+
+Unlike traditional lending protocols where you must manually add collateral or repay debt when prices drop, FCM **uses your yield to maintain position health**. Yield from FYV strategies flows back to ALP automatically, ALP pulls from FYV to repay debt when needed, your position stays healthy without manual intervention, and **you earn yield while protecting yourself from liquidation**.
+
+### 2. Automated Capital Efficiency
+
+FCM maximizes your capital efficiency through automation:
+
+- **Auto-borrowing**: Instantly borrow optimal amount when depositing collateral
+- **Auto-rebalancing**: Maintain target health ratio (1.1-1.5) automatically
+- **Auto-compounding**: FYV strategies reinvest yields
+- **Auto-protection**: Pull from yield to prevent liquidations
+
+### 3. Composable Architecture
+
+Each component can be used independently or together. Use **ALP alone** for traditional lending/borrowing, use **FYV alone** for yield aggregation, or use **FCM together** for the complete yield-generating system.
+
+## Why Use FCM?
+
+### For Yield Seekers
+
+FCM allows you to maximize returns by borrowing against collateral and deploying into high-yield strategies, leverage without liquidation risk as yield protects your positions, set and forget through complete automation, and compound returns as yields reinvest automatically.
+
+### For Conservative Users
+
+FCM provides liquidation protection through yield maintaining position health, flexible health targets allowing you to choose your risk tolerance (1.1-1.5), support for multiple collateral types including FLOW, stFLOW, USDC and more, and complete transparency with all logic on-chain and auditable.
```

Comment:
We mention 'automatically' quite a few times in the overall doc. Also I think its worth pointing out frequency of rebalances or that they can occur many times in a given day